eva-remapping-get-source/src/test/java/uk/ac/ebi/eva/remapping/source/batch/io/VariantContextWriterTest.java

```diff
@@ -152,21 +154,24 @@ public void writeRsId() throws Exception {
     }
 
     private void assertInfo(Long rsId, String infoColumn) {
-        assertInfo(rsId, PROJECT_ACCESSION, TAXONOMY_ACCESSION, infoColumn, CREATED_DATE);
+        assertInfo(SS_HASH, rsId, PROJECT_ACCESSION, TAXONOMY_ACCESSION, infoColumn, CREATED_DATE);
     }
 
-    private void assertInfo(Long expectedRsId, String expectedProject, int expectedTaxonomy, String infoColumn,
-                            String expectedCreatedDate) {
+    private void assertInfo(String expectedSSHash, Long expectedRsId, String expectedProject, int expectedTaxonomy,
+                            String infoColumn, String expectedCreatedDate) {
```

Comment:
Just a comment on the position of passing the parameter infoColumn... So in this case, the infoColumn contains a semi-colon separated values of rs_id, project_id, tax_id and creation date? Do you think that it would be better to place it as the first parameter followed by the rest

Reply:
Addressed...